README.md

```diff
@@ -39,38 +39,40 @@ This vagrant box aims to make it dead simple to start a hashistack and emulate h
 
 ## Content
 
-1. [Description - what & why](#description---what--why)
-    1. [Submodules](#submodules)
-    2. [Services](#services)
-    3. [Why does this exist?](#why-does-this-exist)
-    4. [Installed stack](#installed-stack)
-    5. [Versions](#versions)
-2. [Install prerequisites](#install-prerequisites)
-    1. [General requirements](#general-requirements)
-        1. [Proxy](#proxy)
-    2. [Linux requirements](#linux-requirements)
-    3. [MacOS requirements](#macos-requirements)
-    4. [Windows requirements](#windows-requirements)
-        1. [GNU Make and Git Bash](#GNU-Make-and-Git-Bash)
-        2. [Installation](#installation)
-3. [Build](#build)
-4. [Configuration](#configuration)
-    1. [Default Configuration](#default-configuration)
-    2. [Override default configuration](#override-default-configuration)
-        1. [ENV variables](#option-1---env-variables)
-        2. [Config files](#option-2---config-files)
-5. [Usage](#usage)
-   1. [Starting a plain default box](#option-1-starting-a-plain-default-box)
-      1. [Port collisions](#port-collisions)
-         1. [Shut down the running machine](#shut-down-the-running-machine)
-         2. [Use the `auto_correct` feature to dynamically allocate ports](#use-the-auto-correct-feature-to-dynamically-allocate-ports)
-   2. [Starting a new project based on the template](#option-2-starting-a-new-project-based-on-the-template)
-6. [Test](#test)
-    1. [Local run](#local-run)
-    2. [CI pipeline run](#ci-pipeline-run)
-        1. [CI test configuration](#ci-test-configuration)
-8. [Diagram](#diagram)
-9. [Contribute](#contribute)
+- [- vault development mode](#ullivault-development-modeliul)
```

Comment:
What is this? The section does not exist.

Reply:
Good catch, no clue what happened there. I've removed it